.goreleaser.yml

```diff
@@ -627,14 +610,14 @@ docker_signs:
   - artifacts: all
     stdin: "{{ .Env.COSIGN_PWD }}"
     args: [
-      # Default options
-      "sign",
-      "--key=cosign.key",
-      "${artifact}",
-      "--yes",
-      # Additional options
-      "--recursive"
-    ]
+        # Default options
+        "sign",
+        "--key=cosign.key",
+        "${artifact}",
+        "--yes",
+        # Additional options
+        "--recursive",
+      ]
```

Comment:
PSA, we had to comment this out to fix the release (I have the ticket for turning it back on once we sort out our issues). I suspect this branch has not been rebased yet.

Reply:
Yea I haven't rebased yet